soc/arm/st_stm32/stm32f0/Kconfig.defconfig.stm32f098xx

```diff
@@ -0,0 +1,14 @@
+# ST Microelectronics STM32F098XX MCU
```

Comment:
STM32F098Xc

This file is describing the Xc (32K RAM/256K Flash) variants of STM32F098 SoC.
Please change filename to Kconfig.defconfig.stm32f098Xc.dtsi

Reply:
It's possible to rename the dtsi, but not the Kconfig. The stm32cube headers expect the #define STM32F098XX.